convert2rhel/unit_tests/breadcrumbs_test.py

```diff
@@ -268,3 +270,52 @@ def test_set_target_os(pretend_os):
         "name": "CentOS Linux",
         "version": "7.9",
     } == breadcrumbs.breadcrumbs.target_os
+
+
+@pytest.mark.parametrize(("telemetry_disabled", "telemetry_called"), [(True, 0), (False, 1)])
+def test_disable_telemetry(telemetry_disabled, telemetry_called, monkeypatch):
+    if telemetry_disabled:
+        monkeypatch.setenv("CONVERT2RHEL_DISABLE_TELEMETRY", "1")
+
+    _save_migration_results = mock.Mock()
+    _save_rhsm_facts = mock.Mock()
+
+    monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_migration_results", _save_migration_results)
+    monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_rhsm_facts", _save_rhsm_facts)
+    # Set to true, pretend that user was informed about collecting data
+    breadcrumbs.breadcrumbs._inform_telemetry = True
+
+    breadcrumbs.breadcrumbs.finish_collection()
+
+    assert _save_migration_results.call_count == 1
+    assert _save_rhsm_facts.call_count == telemetry_called
+
+
+def test_user_not_informed_about_telemetry(monkeypatch):
+    _save_migration_results = mock.Mock()
+    _save_rhsm_facts = mock.Mock()
+
+    monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_migration_results", _save_migration_results)
+    monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_rhsm_facts", _save_rhsm_facts)
+    breadcrumbs.breadcrumbs.__init__()  # make sure everything is set to default
```

Comment:
By mocking the previous functions, you won't necessarily need to call __init__(), I think, as it will be always set to `False` at this point, I think.